src/main/java/me/figo/FigoSession.java

```diff
@@ -479,48 +336,40 @@ public void setAccountOrder(List<Account> orderedList) throws FigoException, IOE
      * @param count
      *            limit the number of returned transactions
      * @param offset
-     *            which offset into the result set should be used to determin the first transaction to return (useful in combination with count)
+     *            which offset into the result set should be used to determine the first transaction to return (useful in combination with count)
      * @param include_pending
      *            this flag indicates whether pending transactions should be included in the response; pending transactions are always included as a complete
      *            set, regardless of the `since` parameter
      * @return an array of Transaction objects
      */
-    public List<Transaction> getTransactions(String accountId, String since, Integer count, Integer offset, Boolean include_pending) throws FigoException,
-            IOException {
-        StringBuilder sb = new StringBuilder();
+    public List<Transaction> getTransactions(String accountId, String since, Integer count, Integer offset, PendingTransactions include_pending) throws FigoException, IOException {
+        String path = "";
         if (accountId == null) {
-            sb.append("/rest/transactions?");
```

Comment:
To be fair. On this one we could argue about the resulting bytecode. Your version is more efficient.